src/main/java/competition/subsystems/drive/commands/ArcadeDriveWithKinematicsCommand.java

```diff
@@ -0,0 +1,82 @@
+package competition.subsystems.drive.commands;
+
+import com.google.inject.Inject;
+
+import competition.operator_interface.OperatorInterface;
+import competition.subsystems.drive.DriveSubsystem;
+import xbot.common.command.BaseCommand;
+import xbot.common.math.MathUtils;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.PropertyFactory;
+
+
+public class ArcadeDriveWithKinematicsCommand extends BaseCommand {
+
+    final DriveSubsystem drive;
+    final OperatorInterface oi;
+
+    final DoubleProperty linearVelocityConstantValue;
+    final DoubleProperty angularVelocityConstantValue;
+
+    double linearVelocityConstant = 1; // needs tuning
+    double angularVelocityConstant = 1; // needs tuning
+
+    @Inject
+    public ArcadeDriveWithKinematicsCommand(DriveSubsystem drive, OperatorInterface oi, PropertyFactory propManager) {
+        this.drive = drive;
+        this.oi = oi;
+        this.addRequirements(drive);
+
+        linearVelocityConstantValue = propManager.createEphemeralProperty("Linear Velocity Constant Value", 0.0);
```

Comment:
these need to be persistent (e.g. `propManager.createPersistentProperty`) so that they will stick around once we get good values.